### DISCUSSION TITLE
[v6] correct way to do a masked input?

### DISCUSSION THREAD

**Discussion by piehouserat** [react-hook-form/react-hook-form#1951] [Q&A] [answered]
hi guys,

wondering whats the best way to a implement a masket input with RHF v6, I notice the codesandbox link has been removed in recent commits

**Comment by bluebill1049** [comment, marked as answer]
use Controller with any masking component. There is an example in this codesandbox: https://codesandbox.io/s/react-hook-form-controller-079xx
